barretenberg/cpp/src/barretenberg/translator_vm/translator_verifier.cpp

```diff
@@ -108,7 +108,10 @@ bool TranslatorVerifier::verify_proof(const HonkProof& proof,
     std::array<Commitment, NUM_LIBRA_COMMITMENTS> libra_commitments = {};
     libra_commitments[0] = transcript->template receive_from_prover<Commitment>("Libra:concatenation_commitment");
 
-    auto sumcheck_output = sumcheck.verify(relation_parameters, alpha, gate_challenges);
+    std::array<FF, TranslatorFlavor::CONST_TRANSLATOR_LOG_N> padding_indicator_array;
```

Comment:
Very minor but this pattern appears in a lot of places. Might be nice to just have a class PaddingIndicatorArray that can just be constructed with the right form in a one-liner